dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype-mbeans.properties

```diff
@@ -109,3 +109,6 @@ plugin.trustedProxyEnable.desc=Enables trusted proxies to be used. When specifie
 
 plugin.trustedProxyGroup.name=Override the TrustedProxyGroup property
 plugin.trustedProxyGroup.desc=A comma-separated list of trusted proxies. When specified, this property overrides the value in the web server plug-in configuration file.
+
+plugin.ignoreAffinityRequests.name=Ignore affinity requests
+plugin.ignoreAffinityRequests.desc=Indicates whether affinity requests are ignored when tracking a server's runtime weight for round robin load balancing.
```

Comment:
Is it possible to say specifically what component/ entities are ignoring the affinity requests and what is tracking runtime weight?

For clarity, it's better to avoid elliptical clauses like "when tracking" and include a subject instead. Something like:

`Indicates whether the server ignores affinity requests when it is tracking runtime weight for round-robin load balancing.`

or 

`Indicates whether the server ignores affinity requests when the system is tracking server runtime weight for round-robin load balancing.`

or


`Indicates whether the server ignores affinity requests when its runtime weight is being tracked for round-robin load balancing.`

Reply:
How about:

> Indicates whether the webserver plug-in ignores affinity requests when it is tracking runtime weight for round-robin load balancing.

Reply:
That looks good to me

Reply:
done